tests/integration/cli/test_test.py

```diff
@@ -0,0 +1,46 @@
+import subprocess
+import sys
+
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import busybox_single_app_bundle_fixture
+from dusty.log import client_logger, DustyClientTestingSocketHandler
+
+class TestTestCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestTestCLI, self).setUp()
+        busybox_single_app_bundle_fixture(num_bundles=1, include_tests=True)
+        self.handler = DustyClientTestingSocketHandler()
+        client_logger.addHandler(self.handler)
```

Comment:
interesting question. It does not technically need to be, it probably should be just to help people in the future.